tensorflow/compiler/xla/mlir_hlo/tests/Dialect/mhlo/hlo-legalize-to-linalg.mlir

```diff
@@ -4962,3 +4962,23 @@ func.func @feature_group_count_convolution(%arg0: tensor<2x14x12x2xf64>, %arg1:
     : (tensor<2x14x12x2xf64>, tensor<7x7x1x2xf64>) -> tensor<2x6x8x2xf64>
   return %0 : tensor<2x6x8x2xf64>
 }
+
+// -----
+// The following test is identitical to the previous one, except that the
```

Comment:
@ashay Thank you for adding the test! Sounds like there's a typo here: "identical".